README.md

```diff
@@ -25,68 +25,237 @@ This setup enables the following capabilities:
+
+---
+
+#### Key Observations
+
+- No **Dispatcher cache flushing** is required.
+- Dynamic listings remain up-to-date without re-rendering pages on CMS publishers, reducing the load and cost of maintaining multiple publisher instances.
+
+---
+
+#### Key Takeaway/Value
+
+- **No need to republish** the page with dynamic content.
+- **No cache refresh** is needed; the HTTP server delivers the same homepage.
+- SSI embeds the latest data when the page is requested by visitors.
+- The same approach can deliver **personalized content**.
+
+---
+
+## Part II - Resources Based on Multiple Sources
+
+### Presented Capabilities
+
+- Always up-to-date, **auto-generated sitemaps** integrating multiple source systems (WebSight CMS, PIM).
+- Real-time search serving instant information from source systems.
+
+---
+
+#### Orient the Audience to the Screen
+
+1. Present the **sitemap.xml** file generated based on:
+    - PIM data
+    - CMS pages
+
+2. Present the **search functionality**.
+
+---
+
+#### Scenario
+
+1. **Unpublish product `B075X5TN2K`** (if not already done).
+2. Open the Puresight homepage:
+    - Search for **b075x5tn2k** → Show **no results**.
+3. Open the **sitemap.xml** file:
+    - Search for `b075x5tn2k` → Show **no results**.
+4. Open the PIM and publish the product.
+5. Repeat the search on the Puresight homepage for **b075x5tn2k** → Show the match.
+6. Open **sitemap.xml** → Show the product match.
+7. **Unpublish the blog page**:  
+   "Simplify Your Space: A Guide to Decluttering" from CMS.
+    - Search for **"simplify-your-space"** in sitemap.xml → No results.
+    - Search for **"simplify"** on the site → No results.
+8. **Publish the blog page** again:
+    - Show results in **sitemap.xml** and search.
+
+---
+
+#### Explain What Happened in the Background
+
+- PIM informed StreamX about a **new product**.
+- CMS informed StreamX about a **new page**.
+- StreamX updated the **sitemap** and the **search functionality**.
+
+---
+
+#### Key Takeaway/Value
+
+- **StreamX** ensures that delivery services always use **up-to-date information** from multiple systems.
+
+---
+
+## Part III - Multiple Sources Update the PDP
+
+### Presented Capabilities
+
+- Always up-to-date **Product Detail Page (PDP)** generated using data from multiple sources:
+    - WebSight CMS
+    - PIM
+    - Price script
+    - Reviews script
+
+---
+
+#### Orient the Audience to the Screen
+
+- Present a PDP generated based on multiple sources.
+
+---
+
+#### Scenario
+
+1. Open the PDP from the homepage listing for product `B075X5TN2K`.
+    - Explain which data comes from which system.
+2. Update the price for `B075X5TN2K` → Refresh the PDP and show the updated price.
+3. Add a review for `B075X5TN2K` → Refresh the PDP and show reviews.
+4. Update the product name in PIM for `B075X5TN2K` → Show the updated name on the PDP.
+5. Modify the PDP template in CMS → Refresh the PDP and show the updated content.
```

Comment:
Modify and publish

Reply:
I avoided adding publish info because it would be in every single step. I guess it's obvious that any action needs to be reflected in streamx